ts/screens/authentication/NewOptInScreen.tsx

```diff
@@ -98,7 +98,7 @@ const NewOptInScreen = (props: Props) => {
         }}
       >
         <ContentWrapper>
-          {Dimensions.get("screen").height > 780 && (
+          {Dimensions.get("screen").height > 820 && (
```

Comment:
A modest suggestion: can we use `const` variables instead of setting the number directly? Better if they're global and reusable in other contexts.
I'm pretty sure there will be other screens that need this value.

Reply:
done! :)
Thanks for the suggestion!